python-backend/app/api/mines/permits/permit/models/permit.py

```diff
@@ -75,4 +81,22 @@ def validate_permit_no(self, key, permit_no):
             raise AssertionError('Permit number is not provided.')
         if len(permit_no) > 16:
             raise AssertionError('Permit number must not exceed 16 characters.')
-        return permit_no
\ No newline at end of file
+        return permit_no
+
+
+from marshmallow_sqlalchemy import ModelConversionError, ModelSchema, ModelConverter, fields
+
+
+class PermitSchema(ModelSchema):
```

Comment:
I'm alright with having 2 classes per file but we should call it out as a decision.